src/pybind/mgr/dashboard/frontend/e2e/rgw/users.po.ts

```diff
@@ -39,6 +39,7 @@ export class UsersPageHelper extends PageHelper {
     await this.getTableCell(name).click(); // click on the bucket you want to edit in the table
     await element(by.cssContainingText('button', 'Edit')).click(); // click button to move to edit page
 
+    await this.waitVisibility(this.getBreadcrumb()); // Wait for breadcrumb to be visible
```

Comment:
This should work better, since it will wait for the correct breadcrumb
```suggestion
    await this.waitTextToBePresent(this.getBreadcrumb(), 'Edit'); // Wait for breadcrumb to be visible
```

I'm guessing that `getBreadcrumbText` can be removed and all calls replace with the above...